src/components/navigation/navigation.js

```diff
@@ -1,11 +1,16 @@
 import './navigation.css'
 import {Link} from 'react-router-dom'
 import { useProductContext } from '../../contexts/product-context';
```

Comment:
you can create an `index.js` file and export them all in it and import them here using this file.